### PR TITLE
Add Czech language support 🇨🇿🇪🇺

### PR DESCRIPTION
**Pull request by ademar111190**
Thank you @CS-HODL for the translations 🙏 

How it looks like:

|Home|Drawer menu|Bottom menu|
|---|---|---|
|home|drawer menu|bottom menu|

### CODE REVIEW

File: lib/routes/subswap/swap/widgets/deposit_widget.dart [hunk 1]
```diff
@@ -122,8 +122,6 @@ class _DepositWidgetState extends State<DepositWidget> {
       return texts.invoice_btc_address_warning_without_min_fee_account_connected(
         minSats,
         maxSats,
-        setUpFee,
```
Comment (ademar111190) [inline lib/routes/subswap/swap/widgets/deposit_widget.dart:125]: @ubbabeck If you revert the changes as suggested by @erdemyerebasmaz here https://github.com/breez/breezmobile/pull/1181#pullrequestreview-1558795997
Please update here as well, it is a good example of what I said here: https://github.com/breez/breezmobile/pull/1181#issuecomment-1662156491

Comment (ademar111190) [reply lib/routes/subswap/swap/widgets/deposit_widget.dart]: Reverted as it was reverted on breez translations

**Review by ubbabeck — APPROVED** [review]
Looks good to me 👍

**Review by erdemyerebasmaz — APPROVED** [review]
